server/src/routes/posts.js

```diff
@@ -40,6 +40,28 @@ router.put('/:id', async (req, res) => {
   }
 });
 
+// Get one post (detailed)
+// eslint-disable-next-line no-unused-vars
+router.get('/:id', async (req, res) => {
+  try {
+    const foundPost = await Post.findOne({ _id: req.params.id });
+    const postData = {
+      _id: foundPost._id,
+      title: foundPost.title,
+      body: foundPost.body,
+      upvotes_clap: foundPost.upvotes_clap,
+      upvotes_laugh: foundPost.upvotes_laugh,
+      upvotes_sad: foundPost.upvotes_sad,
+      date_created: foundPost.createdAt,
+      comments: await Comment.find({ _id: { $in: [foundPost.comment_id] } }),
+    };
+
+    res.send(postData);
```

Comment:
Do we need a response status code here?

Reply:
It returns 200 based on the unit tests

Reply:
> Do we want to leave changing to populate for a later group/issue?

Yeah, I've removed my comment also. I realised if we want to support multi nested comments in the future as well, I"m not sure if the populate method will still work.

Reply:
Cool! Should be fine then.